internal/pkg/service/stream/storage/statistics/repository/stats_sum.go

```diff
@@ -1,33 +0,0 @@
-package repository
-
-import (
-	"context"
-	"time"
-
-	"github.com/keboola/keboola-as-code/internal/pkg/service/common/etcdop/iterator"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/common/ptr"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/common/utctime"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/storage/statistics"
-)
-
-// SumStats sums all stats from the iterator.
-func SumStats(ctx context.Context, now time.Time, prefix iterator.DefinitionT[statistics.Value]) (out statistics.Value, err error) {
-	var outReset statistics.Value
-	if err := sumStatsOp(now, prefix, &out, &outReset).Do(ctx).Err(); err != nil {
-		return out, err
-	}
-	return out.Add(outReset), nil
-}
-
-// sumStatsOp sums all stats from the iterator.
-func sumStatsOp(now time.Time, prefix iterator.DefinitionT[statistics.Value], outSum *statistics.Value, outReset *statistics.Value) *iterator.ForEachT[statistics.Value] {
```

Comment:
SumStats was unused, and sumStatsOp was used on 2 places, but it can be simplified too.